docs/introduction/Sidechain-SDK-extension.rst

```diff
@@ -422,21 +420,21 @@ For a successful response, implement SuccessResponse interface with data to be r
     }
     }
 
-In such case API response will be represented in the following JSON format:
+In such a case, the API response will be represented in the following JSON format:
 
   ::
   
     {"result": {“response” : “response from CustomSuccessResponse object”}}
     
-In case of something going wrong and error shall be returned then response shall implement ErrorResponse interface which by default have next functions to be implemented:
+If an error shall be returned, then the response shall implement the ErrorResponse interface, which by default have these functions implemented:
```

Comment:
@zennermac 

If an error is returned, then the response will implement the ErrorResponse interface. The ErrorResponse interface has the following default functions implemented: